docs/cli-preview3/tools/telemetry.md

```diff
@@ -0,0 +1,100 @@
+---
+title: .NET Core Tools Telemetry
+description: .NET Core
+keywords: .NET, .NET Core
+author: richlander
+manager: wpickett
+ms.date: 07/06/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: f2b312bb-f80b-4b0d-9101-93908f06a6fa
+---
+
+# .NET Core Tools Telemetry
+
+The .NET Core Tools include a [telemetry feature](https://github.com/dotnet/cli/pull/2145) that collects usage information. It’s important that the .NET Team understands how the tools are being used so that we can improve them.
+
+The data collected is anonymous and will be published in an aggregated form for use by both Microsoft and community engineers under the [Creative Commons Attribution License](https://creativecommons.org/licenses/by/4.0/).
+
+## Scope
+
+The `dotnet` command is used to launch both apps and the .NET Core Tools. The `dotnet` command itself does not collect telemetry. It is the .NET Core Tools that are run via the `dotnet` command that collect telemetry.
+
+.NET Core commands (telemetry is not enabled):
+
+- `dotnet`
+- `dotnet [path-to-app]`
+
+.NET Core Tools [commands](index.md) (telemetry is enabled), such as:
+
+- `dotnet build`
+- `dotnet pack`
+- `dotnet restore`
+- `dotnet run`
```

Comment:
I thought it did explain already. The telemetry runs on dotnet commands not on the driver. maybe this can be later changed to be more specific. and then later on it goes on to explain what kind of info is being collected.